esprima.js

```diff
@@ -3669,17 +3669,11 @@
 
     function parseExportDeclaration() {
         var backtrackToken, id, declaration = null,
+            peekedToken, sourceElement,
```

Comment:
I like that better, since now it's just used to check if it's an identifier and not for potential rewinding.